src/assets/styles/modules/image-with-text.scss

```diff
@@ -0,0 +1,8 @@
+.image-with-text {
+  display: flex;
+  flex-direction: row;
```

Comment:
We can remove this since `flex-direction` defaults to `row`.

Reply:
👍